x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/job_creator.ts

```diff
@@ -243,6 +243,25 @@ export class JobCreator {
     );
   }
 
+  public set annotations(enable: boolean) {
```

Comment:
If you clone a job, it should copy the `annotations_enabled` setting to the new job. I tried cloning a Single Metric job with the settings

```
  "model_plot_config": {
    "enabled": true,
    "annotations_enabled": false
  },
```
but the annotations switch was enabled in the wizard. It should use the setting from the job you are cloning.

Reply:
Fixed in https://github.com/elastic/kibana/pull/70678/commits/1b54be9b39e990830c85f2506d040e803725dad5

Reply:
Tested latest edits, and this is now working as expected.